packages/shared_preferences/shared_preferences_macos/pubspec.yaml

```diff
@@ -2,23 +2,25 @@ name: shared_preferences_macos
 description: macOS implementation of the shared_preferences plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/shared_preferences/shared_preferences_macos
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+shared_preferences%22
-version: 2.0.2
+version: 2.0.3
```

Comment:
The versions need to be bumped to 2.1.0 still (as noted in the comment about deprecating methods).

Reply:
Ah yes, I forgot about that. You tagged the version for the macos package here though. Shouldn't the minor only be bumped for the linux and windows packages as those are the only ones who had that field deprecated?
Windows & Linux packages have had their pubspec versions and changelog updated.

Reply:
Sorry, that was carelessness as I was scrolling through.